### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by systimotic**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

Hello there human!

The other day, I was looking at the CONTRIBUTING.md file, when I noticed a typo! 😱 _I have to fix this!_
After I opened the file, I ended up fixing and tweaking some other things as well. Here's a summary:
- My editor (Atom) automatically removed some trailing spaces
- There was only a single "I" in the entire file. I changed it to be more consistent.
- The Contributors chat room on Gitter is referenced as "Contributors chat room," "Contributors Chat Room" and "Contributors room", sometimes as a link, sometimes not. I edited some cases to be more consistent.
- The "How to find the code in the Free Code Camp codebase to fix/edit?" was in the PR section. I moved it near "Make changes" to hopefully be easier to find and harder to miss.
- With the current state of the project, after making changes to the seed files, you need to run `node seed` for them to be visible. This was never mentioned in this file. I added a section that talks about this. I assume it wasn't there because it previously updated automatically. I hope this helps resolve a common confusion among new contributors.
- Previously, the test section mentioned running a different command on Windows to test. This has been fixed (ref #10885), so I removed that part.
- The linting section mentioned that lines shouldn't be longer than 80 characters because of jsonlint. I'm not sure why or how that ever worked. As the name might suggest, jsonlint only lints the JSON. The linting talked about in this section only lints the .js and .jsx files. There is no maximum line length in the JSON files. I have tested this, it doesn't break. I removed this section because it seemed to be false information. 😅 Apart from that, it seems weird to me to only detail a single rule out of alllll of them.
- These last two are really nitpicking. I fixed the casing on a username. I also switched the order of two items in an unordered list to match the headings that followed it.

I know it's not such a great idea to stuff many different things into a single PR, but once I got started, I couldn't stop myself 😊

Have a great day! 🔥

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -156,7 +156,6 @@ You should have [ESLint running in your editor](http://eslint.org/docs/user-guid
 
 > Please do not ignore any linting errors, as they are meant to **help** you and to ensure a clean and simple code base.
 
-Make sure none of your JavaScript is longer than 80 characters per line. The reason we enforce this is because one of our dependent NPM modules, [jsonlint](https://github.com/zaach/jsonlint), does not fully support wildcard paths in Windows.
```
Comment (Bouncey) [inline CONTRIBUTING.md]: Why is this removed?

It's still a style rule for the codebase

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -205,24 +204,35 @@ gulp
...
+that you were looking forward to edit. Always feel free to reach out to the chat
+room when you are not certain of any thing specific in the code.
+
+#### Changes to the seed files
+If you made changes to any file in the `/seed` directory, you need to run
 ```shell
-$ npm test
+$ node seed
 ```
+in order for the changes to be activated.
```
Comment (Bouncey) [inline CONTRIBUTING.md]: ~~in order for the changes to be activated.~~
in order to see the changes.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -243,16 +253,7 @@ accepted by Free Code Camp proper.
...
-
-#### How to find the code in the Free Code Camp codebase to fix/edit?
-
-The best way to find out any code you wish to change/add or remove is using
-the GitHub search bar at the top of the repository page. For example, you could
-search for a challenge name and the results will display all the files along
-with line numbers. Then you can proceed to the files and verify this is the area
-that you were looking forward to edit. Always feel free to reach out to the chat
-room when you are not certain of any thing specific in the code.
+related issues on our [Contributors chat room](https://gitter.im/FreeCodeCamp/Contributors).
```
Comment (Bouncey) [inline CONTRIBUTING.md]: ~~related issues on our~~
related issues in our

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -266,8 +267,8 @@ fork and re-fork.
 
 There are two methods of creating a pull request for Free Code Camp:
 
--   Editing files via the GitHub Interface
 -   Editing files on a local clone
```
Comment (Bouncey) [inline CONTRIBUTING.md]: ~~Editing files on a local clone~~
Editing files on a local clone (recommended)

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bouncey) [inline CONTRIBUTING.md]: You can remove the part about `jsonlint` :+1:

Comment (systimotic) [inline CONTRIBUTING.md]: @Bouncey So what's the final verdict on this? 
My reasons for removing it: 
- This rule seemed to have nothing to do with jsonlint.
- There are a bunch of rules, but this is the only one that is mentioned. That could be misleading (it might be seen as the only important one).

**Comment by systimotic** [comment]
Thank you for the review @Bouncey! They're small changes, but they truly make the document easier to read.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bouncey) [inline CONTRIBUTING.md]: I think it is more historical, the 80char rule tripped up many contributor because we weren't running `npm test` before pushing a commit, which annoying broke the travis build.

We couldn't default everyone to  `npm test` because of the jsonlint/windows issue, so we mentioned the 80char rule here instead 

I guess if we are all now doing `npm test`, it can be removed :+1:

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
## LGTM :+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (systimotic) [inline CONTRIBUTING.md]: Cool, thanks for the clarification!
